src/vorta/config.py

```diff
@@ -1,14 +1,14 @@
 import os
 from pathlib import Path
-import appdirs
+import platformdirs
 
 APP_NAME = 'Vorta'
 APP_AUTHOR = 'BorgBase'
 APP_ID_DARWIN = 'com.borgbase.client.macos'
-dirs = appdirs.AppDirs(APP_NAME, APP_AUTHOR)
-SETTINGS_DIR = dirs.user_data_dir
-LOG_DIR = dirs.user_log_dir
-CACHE_DIR = dirs.user_cache_dir
+dirs = platformdirs.PlatformDirs(APP_NAME, APP_AUTHOR)
+SETTINGS_DIR = dirs.user_data_path
+LOG_DIR = dirs.user_log_path
+CACHE_DIR = dirs.user_cache_path
 TEMP_DIR = os.path.join(CACHE_DIR, "tmp")
 PROFILE_BOOTSTRAP_FILE = Path.home() / '.vorta-init.json'
 
```

Comment:
What about simplifying the existing code like that?

```py
# ensure directories exist
for dir in (SETTINGS_DIR, LOG_DIR, CACHE_DIR, TEMP_DIR):
    dir.mkdir(parents=True, exist_ok=True)
```

Reply:
I tried simplifying as per your suggestion but **TEMP_DIR** return a *str* so I have converted it into *path* is that fine?

Reply:
That is what I intended.